microsoft-edge/webview2/release-notes/index.md

```diff
@@ -17,18 +16,140 @@ These Release Notes provide information about new features and bug fixes that ar
 the templates to copy for incoming sections are in ./includes/templates.md
 
 this webpage covers the most recent 6 months; periodically move oldest h2 sections from bottom of present file to archive.md.  eg covers:
+Oct 2025
 Sep 2025
 Aug 2025
 Jul 2025
 Jun 2025
 May 2025
-Apr 2025
 
 moved to archive:
-Mar 2025
+Apr 2025
 -->
 
 
+<!-- Oct 2025 Release SDK -->
+<!-- ====================================================================== -->
+## 1.0.3537.50
+
+Release Date: October 7, 2025
+
+[NuGet package for WebView2 SDK 1.0.3537.50](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3537.50)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 141.0.3537.50 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+No additional APIs have been promoted from Phase 2: Stable in Prerelease, to Phase 3: Stable in Release, in this Release SDK.
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+This Release SDK includes the following bug fixes.
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Fixed an issue where a "local access network alert" popup was sometimes hidden.
```

Comment:
check wording vs. the orig wording, which was:

* Hiding local access network alert pop up

@Ju-MsGit @vootlasaichandu